README.md

```diff
@@ -1,19 +1,75 @@
-# sturdy-memory
-A simple inventory management system for the Mongoose system of SV Sticky. 
+# Stocky
 
-Fallacious rooster has been created as per request of the 18'th Commissioner of Internal Affairs, in order to keep track of the "black box" the loss of products in the Mongoose sometimes can be.
+**Stocky** is an inventory management app+API+database built with Python, FastAPI, and PostgreSQL. This project is designed to be run entirely inside a Docker-based development container, so all you need is [Docker](https://www.docker.com/) and [Visual Studio Code](https://code.visualstudio.com/) with the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).
 
-## Development 
-The guide below will tell you roughly how to get started with working on sturdy-memory.
 
-## Koala
-Sturdy-memory uses Koala for comparing succesful transactions with the counted amount of the board member checking the amount.
+## Table of Contents
 
-## Server setup
-The server is written in.....? (To be determened)
+- [Getting Started](#getting-started)
+- [Running the Application](#running-the-application)
+- [Contributing](#contributing)
+- [License](#license)
 
-## Frontend setup
-The frontend is written in....? (To be determened)
 
-# License
-GNU GPL v3
\ No newline at end of file
+
+## Getting Started
+
+### Prerequisites
+
+- [Docker](https://www.docker.com/)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [VS Code Dev Containers Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+### Setup with VS Code Dev Container
+
+1. **Clone the Repository:**
+
+   ```bash
+   git clone https://github.com/svsticky/sturdy-memory.git`
+   cd sturdy-memory
+   ```
+
+2. **Open in VS Code:**
+
+   Open the project folder in VS Code. If you have the Dev Containers extension installed, you will be prompted to reopen the project in a container.  
+   Alternatively, open the Command Palette (`Ctrl+Shift+P` or `Cmd+Shift+P`), then select **Remote-Containers: Reopen in Container**.
+
+3. **Dev Container Initialization:**
+
+   The provided `devcontainer.json` (inside the `.devcontainer` folder) will automatically set up your environment, installing all necessary dependencies. No local Python or PostgreSQL installation is required.
+
+
+
+## Running the Application
+
+1. Copy `sample.env` to `.env`. Make sure all values are properly populated. TODO: add explenation of values to populate.
```

Comment:
This TODO should move to the sample.env file imo. I believe it's neater that the .env file itself explains how it can be filled.